assembl/configs/base_env.rc

```diff
@@ -167,4 +167,6 @@ maxretry = 10
 
 # To isolate themes, each have been put in a bucket. Declare where the bucket is for access
 # ie. https://<bucket-url>, or on local machine: /opts/assembl_themes/<bucket>/
-theme_base =
\ No newline at end of file
+theme_base =
+minimum_password_length = 9
+password_required_classes = '{"[a-z]": {"en": "lower-case letter"}, "[A-Z]": {"en": "upper-case letter"}, "\\d": {"en": "digit"}, "\\W": {"en": "special character"}}'
```

Comment:
The regex escape looks off. Have you tried putting `\\d` instead? I'm certain that's what causes an issue with the tests not passing.

Reply:
I suppose you mean `\d`. I did, but then the regex is wrong when I generate the local.ini, because it needs to be escaped there. And if I remove the single quotes, it doesn't work neither...
I don't know how to fix it right now...